services/idp/public/index.html

```diff
@@ -12,6 +12,6 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
-    <main id="root" class="oc-login-bg" data-path-prefix="__PATH_PREFIX__" passwort-reset-link="__PASSWORD_RESET_LINK__"></main>
+    <main id="root" data-path-prefix="__PATH_PREFIX__" passwort-reset-link="__PASSWORD_RESET_LINK__" data-bg-img="__BG_IMG_URL__"></main>
```

Comment:
Since you removed the class ‘oc-login-bg’, will there be an issue while rendering without a background image provided via config vars ?

Reply:
You got it, it's handled in the `tsx`. We add the class there instead on the root component, and override with the custom image if configured.